### PR TITLE
feat(route): 添加济南市卫建委获取国家医师资格考试路由

### PR DESCRIPTION
**Pull request by tzjyxb**
## 该 PR 相关 Issue / Involved Issue

Close #

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/gov/jinan/healthcommission/medical_exam_notice
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/gov/jinanhealthcommission/medical_exam_notice.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            item = $(item).text();
+
+            // 解析上一步中的html
+            const html = cheerio.load(item);
+
+            const title = html('td[width="620"] a').attr('title');
+            const link = html('td[width="620"] a').attr('href');
+            const date = timezone(parseDate(html('td[width="100"]').text()), +8);
+            return {
+                title: title,
```
[the remark this answers is not in the document]
Comment (tzjyxb) [reply lib/v2/gov/jinanhealthcommission/medical_exam_notice.js]: fixed: Expected property shorthand

File: lib/v2/gov/jinanhealthcommission/medical_exam_notice.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                description: title,
+                link: link,
```
[the remark this answers is not in the document]
Comment (tzjyxb) [reply lib/v2/gov/jinanhealthcommission/medical_exam_notice.js]: fixed: Expected property shorthand

File: docs/government.md [hunk 3]
```diff
@@ -470,6 +470,12 @@ pageClass: routes
 
 <Route author="Jeason0228" example="/gov/hunan/notice/all" path="/gov/hunan/notice/:type"  :paramsDesc="['all=全部，cg=采购公告,zb=中标公告,fb=废标公告,ht=合同公告,gz=更正公告,zz=终止公告,qt=其他公告']" />
 
+## 济南市卫生健康委员会
+
+### 获取国家医师资格考试通知
+
+<Route author="tzjyxb" example="/gov/jinanhealthcommission/medical_exam_notice" path="/gov/jinanhealthcommission/medical_exam_notice" radar="1"/>
```
Comment (TonyRL) [inline docs/government.md:477]: Please update path to `/gov/jinan/...`

File: lib/v2/gov/jinanhealthcommission/medical_exam_notice.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const baseUrl = 'https://jnmhc.jinan.gov.cn';
+    const url = 'https://jnmhc.jinan.gov.cn/module/web/jpage/dataproxy.jsp?page=1&webid=28&path=/&columnid=14418&unitid=18878&webname=%25E6%25B5%258E%25E5%258D%2597%25E5%25B8%2582%25E5%258D%25AB%25E7%2594%259F%25E5%2581%25A5%25E5%25BA%25B7%25E5%25A7%2594%25E5%2591%2598%25E4%25BC%259A&permissiontype=0';
```
Comment (TonyRL) [inline lib/v2/gov/jinanhealthcommission/medical_exam_notice.js:8]:
```suggestion
    const res = await got('https://jnmhc.jinan.gov.cn/module/web/jpage/dataproxy.jsp', {
        searchParams: {
            page: 1,
            webid: 28,
            path: '/',
            columnid: 14418,
            unitid: 18878,
            webname: '济南市卫生健康委员会',
            permissiontype: 0,
        },
    });
```

File: docs/government.md (hunk 3 above)
Comment (tzjyxb) [reply docs/government.md]: Thank you. The path has been updated to `/gov/jinan/healthcommission...`

File: lib/v2/gov/jinanhealthcommission/medical_exam_notice.js (hunk 4 above)
Comment (tzjyxb) [reply lib/v2/gov/jinanhealthcommission/medical_exam_notice.js]: Thank you. The suggested chage has bee made.

**Pull request closed without merging**